jot/JotTextEditView.m

```diff
@@ -14,6 +14,8 @@
 
 #import "UIColor+Jot.h"
 
+static const NSUInteger kCharachterLimit = 140;
```

Comment:
spellcheck. Also is this consistent across iOS and Android?

Reply:
Good catch! The updated character count limit to `140` was something we've been intending to do for a while now. Synced with android + design on this too